src/app/App.js

```diff
@@ -3,10 +3,29 @@ import '../assets/App.css';
 import { BrowserRouter as Router } from 'react-router-dom';
 
 import Header from './components/Header';
+import Main from './components/Main';
+
+//firebase
+import firebase from 'firebase';
+// Required for side-effects
+require("firebase/firestore");
 
-import Main from './components/Main'
 
 class App extends Component {
+
+  componentDidMount() {
+    // Initialize Firebase
+    var config = {
+      apiKey: "AIzaSyDSPa0f8VuxemCZG3J8Xjvax4skoA0OUnw",
```

Comment:
This is exactly what should not be on Github. @cocoral, please don't use this database for anything important 😈